target/rv32imcb/testlist.yaml

```diff
@@ -40,6 +40,7 @@
   gen_test: riscv_rand_instr_test
   gen_opts: >
     +disable_compressed_instr=1
+    +enable_b_extension=1
```

Comment:
Please create a new test entry for bit manipulation rather than changing existing test

Reply:
Updated, PTAL. Thanks